enc/unicode/15.0.0/name2ctype.h

```diff
@@ -41768,16 +41769,17 @@ uniname2ctype_hash (register const char *str, register size_t len)
       6099, 6099, 6099, 6099, 6099, 6099, 6099, 6099, 6099, 6099,
       6099, 6099, 6099, 6099, 6099, 6099, 6099, 6099, 6099, 6099,
       6099, 6099, 6099, 6099, 6099, 6099, 6099, 6099, 6099, 6099,
-#ifndef USE_UNICODE_AGE_PROPERTIES
```

Comment:
the following diff was in the gperf output for me even before i updated enc-unicode.rb

Reply:
This seems cased by used `diff`.
What `diff` did you use?

Reply:
Confirmed that the following hunks don't exist when using GNU diffutils 3.8.

Reply:
thanks for checking!

i hadn't noticed that `enc-unicode.rb` uses `diff` internally and was using FreeBSD diff. GNU diff 3.**9** also produces slightly different output btw.

Reply:
GNU diff 3.9 has a bug that `-D` option does not work at all.